ci/submission-handlers.yml

```diff
@@ -17,13 +17,6 @@ deploy-dev-handlers:
     - build-test
   script: gradle deployHandlers -Penv=dev -PdeployPath=/nfs/biostudies/.adm/apps/handlers/dev
 
-deploy-dev-handlers:
```

Comment:
why we are removing this module deployment?

Reply:
These lines are duplicated